test/e2e/instrumentation/monitoring/custom_metrics_stackdriver.go

```diff
@@ -141,14 +141,11 @@ func testAdapter(f *framework.Framework, kubeClient clientset.Interface, customM
 	if err != nil {
```

Comment:
I can't comment on lines outside of this change so adding here:

I believe in line 124 you should be checking that you actually receive both metrics. Now you only check that you don't get extra metrics.

Reply:
Done.